containers/hugo/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,34 @@
+{
+	"name": "Hugo",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"args": {
+			// Update 'VARIANT' to pick hugo variant.
+			// Example variants: hugo, hugo_extended
+			// Rebuild the container if it already exists to update.
+			"VARIANT": "hugo",
+			// Update latest to pick a specific hugo version.
+			// Example versions: latest, 0.73.0, 0,71.1
+			// Rebuild the container if it already exists to update.
+			"VERSION": "latest",
+		}
+	},
+	// Set *default* container specific settings.json values on container create.
+	"settings": {
+		"terminal.integrated.shell.linux": "/bin/ash"
+	},
+	// Add the IDs of extensions you want installed when the container is created.
+	"extensions": [],
```

Comment:
Would it make sense to install an extension for toml support for config.toml?

Reply:
Makes sense. I've added Better-Toml. Updating Now.